crates/credentials/src/presentation_definition.rs

```diff
@@ -262,7 +262,7 @@ mod tests {
             TestVectorFile::load_from_path(path);
 
         for vector in vectors.vectors {
-            let presentation_definition = vector.input.presentation_definition;
+        let presentation_definition = vector.input.presentation_definition;
```

Comment:
Testing if this works... will undo once pipeline proves failure

Reply:
Alright and that's successfully passing now ✅  https://github.com/TBD54566975/web5-rs/actions/runs/9067119392/job/24911514344?pr=203